storage/database/database_test.go

```diff
@@ -104,7 +104,7 @@ var test_values = []string{"a", "1251", "\x00123\x00"}
 var testDatabases = []func() (Database, func()){newTestLDB, newTestBadgerDB, newTestMemDB, newTestDynamoS3DB}
 
 // TestDatabase_PutGet tests the basic put and get operations.
-func TestDatabase_PutGet(t *testing.T) {
```

Comment:
Why did these four database test failed? 
I think DynamoS3DB is in `testDatabases`.
Why don't you just remove TestDynamoS3DB from `testDatabases` and try again?

Reply:
Thank you for your review.
You are right. I will change to enable the rest of the databases except for the DynamoDB.